ui-components/src/page_components/listing/listing_sidebar/QuantityRowSection.tsx

```diff
@@ -39,7 +39,13 @@ const QuantityRowSection = ({ quantityRows, strings }: QuantityRowSectionProps)
       </Heading>
       <div>
         {strings.description && (
-          <p className="text-tiny text-gray-800 pb-3">{strings.description}</p>
+          <div className="text-tiny text-gray-800 pb-3">
+            {typeof strings.description === "string" ? (
```

Comment:
By having the description be `string | React.ReactNode` numbers are valid since React.ReactNode considers string and number to be valid. So if the description is a number the `<p>` tag is not getting added. I don't think it's an issue from a visual perspective, but it might not be what you want.

Reply:
A number isn't an expected use case for this string - I think the intention of `React.ReactNode` here is that route is allowing consumers to have full control, so if they decide to pass anything of that type, it's their responsibility to do it appropriately (although, we don't have any docs yet in uic about guidelines to help people do this, which I think Jesse intends in the future).